src/components/RepositoryGoals.js

```diff
@@ -45,6 +45,8 @@ function RepositoryGoals({user}) {
     dispatch({type: "UPDATE", payload: updatedRepos});
   };
 
+  const data = repository.data && JSON.parse(repository.data.text)
+
```

Comment:
@yg this check here, to prevent the stars from showing if there is no data.json. 

I plan to do a similar check to sync the user's goals data.